### PR TITLE
Update query to use hardcoded values for colocated solvers

### PR DESCRIPTION
**Pull request by bram-vdberg**
This PR addresses issue #12 and updates the CIP-48 query to use hard-coded values for colocated drivers. 

This change also updates the logic for calculating the 3 month grace period for colocated solvers. Now, the logic only sets the grace period if they are manually marked as being in a reduced bonding pool.

Currently hardcoding all colocated solvers. Updates have been made to the CMS to allow us to get this information in a GitHub action, but since that's not live yet I'm adding these manually.

### CODE REVIEW

File: cowprotocol/accounting/rewards/mainnet/service_fee_query_4017925.sql [hunk 1]
```diff
@@ -1,5 +1,15 @@
 with
 
+-- Add colocated solvers here
+colocated_solvers as (
+    select
+        'prod-Barter' as solver_name,
```
Comment (harisang) [inline cowprotocol/accounting/rewards/mainnet/service_fee_query_4017925.sql:6]: note we also have barn-Barter and prod-Copium_Capital that are colocated

Comment (bram-vdberg) [reply cowprotocol/accounting/rewards/mainnet/service_fee_query_4017925.sql:6]: Thanks, I was hoping to try filling them automatically using the CMS. If I can't set that up on time before Monday I'll just add those manually here and set this PR as ready to review.

**Comment by harisang** [comment]
I took the liberty to add an alternative query that is simpler, at least to me, to parse. It is definitely doing redundant things and recomputing things here and there, but i had a bit of a hard time parsing the original query, so this is why i decided to do this.

I am not suggesting we should use the new one, but would be good to have a look at both, proof-read them and then make a call

**Comment by harisang** [comment]
One thing that needs to happen is that the original query should better invoke query 1541516 as black box, even if that means repeating some calculations. @bram-vdberg would be good to check how much this would simplify your query

**Comment by harisang** [comment]
Merging now as we need to execute the payouts tomorrow. Will open an issue to simplify the query

**Pull request merged**